src/mlir/verona-mlir.cc

```diff
@@ -142,47 +143,19 @@ int main(int argc, char** argv)
         return 1;
       }
       // Parse AST file into MLIR
-      try
-      {
-        gen.readAST(m->ast);
-      }
-      catch (std::runtime_error& e)
-      {
-        std::cerr << "ERROR: cannot convert Verona file " << inputFile
-                  << " into MLIR" << std::endl
-                  << e.what() << std::endl;
-        return 1;
-      }
-      break;
+      check(gen.readAST(m->ast));
     }
+    break;
```

Comment:
Small nit, feel free to ignore. I prefer the `break`s in a switch statement be at a consistent level of indentation.

Reply:
I'm using clang-format and that's what it chose. If we change that, the clang-format test will fail in CI.